### PR TITLE
fix(e2e tests): missing new eigenda-client required config fields

### PR DESCRIPTION
**Pull request by samlaf**
was missing ethrpc and svcmanageraddr

Also realized `pull_request_target` was super wrong...... reverted that commit.
pull_request_target by default checks out the target branch! so all the tests were being done against main branch and not the PR branch..

## Fixes Issue

Fixes #

### CODE REVIEW

File: e2e/setup.go [hunk 1]
```diff
@@ -183,18 +183,21 @@ func TestSuiteConfig(testCfg *Cfg) server.CLIConfig {
 		panic(err)
 	}
 
+	svcManagerAddr := "0xD4A7E1Bd8015057293f0D0A557088c286942e84b" // incompatible with non holeskly networks
```
Comment (epociask) [inline e2e/setup.go:186]: Knit - could just say holesky only

**Review by epociask — DISMISSED** [review]

File: e2e/setup.go (hunk 1 above)
Comment (samlaf) [reply e2e/setup.go]: [f2fc7b6](https://github.com/Layr-Labs/eigenda-proxy/pull/196/commits/f2fc7b6eebb4762169a116396cb3d99518580463)

File: go.mod [hunk 2]
```diff
@@ -5,7 +5,9 @@ go 1.22
 toolchain go1.22.0
 
 require (
-	github.com/Layr-Labs/eigenda v0.8.5-0.20241028201743-5fe3e910a22d
+	// TODO: merge https://github.com/Layr-Labs/eigenda/pull/849 first
+	// and then update this dep to the merged commit.
+	github.com/Layr-Labs/eigenda v0.8.5-0.20241030205712-b301232f7804
```
Comment (samlaf) [inline go.mod:10]: TODO

Comment (samlaf) [reply go.mod]: Done: [adf580f](https://github.com/Layr-Labs/eigenda-proxy/pull/196/commits/adf580f21b35f0c4174a9e54b19ccc894ea1d6cb)

**Comment by samlaf** [comment]
Merged the upstream https://github.com/Layr-Labs/eigenda/pull/849, and updated eigenda dep to eigenda master head. Merging without waiting to unblock here.

**Pull request merged**